.azure-pipelines/template/cleanup.yml

```diff
@@ -0,0 +1,14 @@
+steps:
+- script: |
```

Comment:
there is already a cleanup.yml file under .azure-pipelines, why do we need another one?